test/unit/mssqlProtocolHandler.test.ts

```diff
@@ -0,0 +1,189 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as TypeMoq from "typemoq";
+import { assert } from "chai";
+import { MssqlProtocolHandler } from "../../src/mssqlProtocolHandler";
+import SqlToolsServiceClient from "../../src/languageservice/serviceclient";
+import { Uri } from "vscode";
+
+suite("MssqlProtocolHandler Tests", () => {
```

Comment:
Thanks for adding tests. 

Maybe add tests that test invalid values for some attributes. That might increase the code coverage for the protocolhandler file.

-Invalid boolean value
-Invalid numberic value
-Invalid parameter name